### ISSUE TITLE
On this page you reference start and stop in the lookup table but the table has stop twice and no start

### ISSUE THREAD

**Issue by precisioninfinity** [stjude/learngenomics.dev#7]
https://learngenomics.dev/docs/biological-foundations/translation

"As shown in the amino acid lookup table below, other codons specify that translation should start or stop at a particular position." but in the table it says 'STOP' twice with no 'START'

**Comment by claymcleod** [comment]
This is a good point! The reason for this is that the START codon is [always encoded as methionine in humans](https://en.wikipedia.org/wiki/Start_codon). This corresponds to "AUG" in the table. We also say later

> the translation machinery correctly reads a start codon ("AUG" or Methionine)

and

> where the functional or open reading frame is one that begins with the start codon "AUG" (resulting in the Methionine)

I think the reason for the confusion is that the START codon encodes for an amino acid to be added to the chain, so we have something to put there in the table, whereas the STOP codon [does not encode for any amino acid to be added to the chain](https://en.wikipedia.org/wiki/Stop_codon):

> Most codons in messenger RNA correspond to the addition of an [amino acid](https://en.wikipedia.org/wiki/Amino_acid) to a growing [polypeptide](https://en.wikipedia.org/wiki/Polypeptide) chain, which may ultimately become a protein; stop codons signal the termination of this process by binding [release factors](https://en.wikipedia.org/wiki/Release_factor), which cause the [ribosomal](https://en.wikipedia.org/wiki/Ribosome) subunits to disassociate, releasing the amino acid chain.

Ultimately, I agree it could be more clear. I probably will add `(START)` to the table alongside the `met` that is already there.

**Comment by precisioninfinity** [comment]
Yeah, I like that, I think if you had `(START)` in the table alongside `met`, then I wouldn't have even questioned it.
